packages/@ionic/cli/src/lib/project/index.ts

```diff
@@ -508,7 +508,7 @@ export abstract class Project implements IProject {
     let pkgPath: string | undefined;
 
     try {
-      pkgPath = pkgName ? require.resolve(`${pkgName}/package`, { paths: compileNodeModulesPaths(this.directory) }) : this.packageJsonPath;
+      pkgPath = pkgName ? require.resolve(`${pkgName}/package.json`, { paths: compileNodeModulesPaths(this.directory) }) : this.packageJsonPath;
```

Comment:
Will this break for developers on older versions of Ionic?

Reply:
I tested on Ionic 6 and 7 for Angular, React and Vue and it continued to log the appropriate message (no warning displayed). I don't have an Ionic 5 project to test against, but think we are good as long as the CLI supports the current supported versions of Ionic Framework. It seems to support the changes in v7 with the `exports` object in the package.json as well as the legacy format from earlier versions of ng-packagr.

Reply:
Do we need to make the same change in `requirePackageJson` below?

Reply:
I guess it wouldn't hurt - I'm not sure what the part of the implementation accounts for. It looks to be potentially for Angular workspace configurations (having many apps in one project).